### PR TITLE
Clean up compiler warnings

### PR DESCRIPTION
**Pull request by kevinlahey**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
+CC = clang
 LDLIBS = -lcurses
-CFLAGS = -g
+CFLAGS = -g -Wall
```
Comment (vishwas-ftx) [inline Makefile:3]: These options have to move to **configure.ac** after auto* tools merge.

File: sgxtop.c [hunk 2]
```diff
@@ -460,8 +461,9 @@ void enclaves_report(struct enclaves *enclaves)
 int main(int argc, char **argv)
 {
 	struct stats old, new;
-	struct enclaves *enclaves = enclaves_create(101);
+	struct enclaves *enclaves = NULL;
 
+	enclaves = enclaves_create(101);
```
Comment (vishwas-ftx) [inline sgxtop.c]: Can we create a Constant (MAX_ENCLAVES ?) for 101 ?

File: Makefile (hunk 1 above)
Comment (kevinlahey) [reply Makefile:3]: We're deleting all of Makefile, anyway, right?

File: sgxtop.c (hunk 2 above)
Comment (kevinlahey) [reply sgxtop.c]: Sure, good call.  This is actually the count of hash buckets.  It's kind of a big number for that purpose.

File: Makefile (hunk 1 above)
Comment (vishwas-ftx) [reply Makefile:3]: yes. Need to bring these into configure.ac

**Review by vishwas-ftx — APPROVED** [review]

**Pull request merged**